MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,1085 @@
+<td>ValidMessage</td>
+<td>Gets or sets a string which will get displayed when user enters valid value for the data field as success message.</td>
+</tr>
+</table>
+
+### DataFormDateRangeAttribute
+
+<table>
+
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[MinDay](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DateRangeAttribute.html#Syncfusion_XForms_DataForm_DateRangeAttribute_MinDay)'| markdownify }}</td>
+<td>MinimumDate</td>
+<td>Gets or sets the minimum date that users can assign to the date editor.</td>
+</tr>
+
+<tr>
+<td>{{'[MaxDay](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DateRangeAttribute.html#Syncfusion_XForms_DataForm_DateRangeAttribute_MaxDay)'| markdownify }}</td>
+<td>MaximumDate</td>
+<td>Gets or sets the maximum date that users can assign to the date editor.</td>
+</tr>
+
+<tr>
+<td>{{'[MinMonth](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DateRangeAttribute.html#Syncfusion_XForms_DataForm_DateRangeAttribute_MinMonth)'| markdownify }}</td>
+<td>Nil</td>
+<td>Gets or sets the minimum month of a date.</td>
+</tr>
+
+<tr>
+<td>{{'[MaxMonth](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DateRangeAttribute.html#Syncfusion_XForms_DataForm_DateRangeAttribute_MaxMonth)'| markdownify }}</td>
+<td>Nil</td>
+<td>Gets or sets the maximum month of a date.</td>
+</tr>
+
+<tr>
+<td>{{'[MinYear](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DateRangeAttribute.html#Syncfusion_XForms_DataForm_DateRangeAttribute_MinYear)'| markdownify }}</td>
+<td>Nil</td>
+<td>Gets or sets the minimum year of a date.</td>
+</tr>
+
+<tr>
+<td>{{'[MaxYear](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DateRangeAttribute.html#Syncfusion_XForms_DataForm_DateRangeAttribute_MaxYear)'| markdownify }}</td>
+<td>Nil</td>
+<td>Gets or sets the maximum year of a date.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>DisplayFormat</td>
+<td>Gets or sets the attribute date display format of date editor.</td>
+</tr>
+</table>
+
+### DataFormValueConverterAttribute 
+
+<table>
+
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[ConverterType](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.ConverterAttribute.html#Syncfusion_XForms_DataForm_ConverterAttribute_ConverterType)'| markdownify }}</td>
+<td>ConverterType</td>
+<td>Gets the Converter type.</td>
+</tr>
+</table>
+
+## Enums
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[CommitMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.CommitMode.html)'| markdownify }}</td>
+<td>DataFormCommitMode</td>
+<td>Defines the constants for CommitMode that specifies when the value entered by user should be saved to the data object.</td>
```

Comment:
<td>Defines the constants for the CommitMode that specify when the value entered by the user should be saved to the data object.</td>